main.cpp

```diff
@@ -391,4 +408,17 @@ int main()
     bar.turnDown(6);
 
     chris.changeSize(1);
+
+    std::cout << "These shoes are in " << blazer.condition << " condition" << std::endl;
+
+    std::cout << "The Wulitzer has " << red.numKeys << " keys" << std::endl;
+
+    std::cout << "This Bass costs $" << jaguar.cost << std::endl;
+
+    std::cout << "There are " << bar.members << " members in the band" << std::endl;
+
+    blazer.print();
+    red.print();
+    jaguar.print();
+    bar.print();
```

Comment:
Call the function that produces the same output DIRECTLY AFTER the cout statement in main().
Take a look at `Example::main()` if needed.
- [ ] revise